### PR TITLE
[ENG-27351] new fields limit

### PR DESCRIPTION
**Pull request by hannah-sk**
### Related issue:
[ENG-27351]

### Changes

* Added the new limit for Real-Time Events GQL query fields in EN and PT.
* Added note for users to check the Limits doc in EN and PT.

[ENG-27351]: https://aziontech.atlassian.net/browse/ENG-27351?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/content/docs/en/pages/api-graphql/limits/limits.mdx [hunk 1]
```diff
@@ -13,13 +13,19 @@ To keep **GraphQL API** working with enhanced performance, the API has predeterm
...
+## Selected fields limit Real-Time Metrics GraphQL API
 
-When you’re querying for datasets’ fields, the maximum value of selected fields is of *35 fields per query*. If you surpass the fields' limit, the API will return an Error Response message informing the issue.
+The maximum value of selected fields for a dataset is of *35 fields per query* in the Real-Time Metrics GQL. If you surpass the fields' limit, the API will return an error response message informing the issue.
+
+---
+
+## Selected fields limit Real-Time Events GraphQL API
+
+The maximum value of selected fields for a dataset is of *10 fields per query* in the Real-Time Events GQL. If you surpass the fields' limit, the API will return an error response message informing the issue. If you need to analyze more fields, create different queries.
```
Comment (MarianaAguilera) [inline src/content/docs/en/pages/api-graphql/limits/limits.mdx:28]:
```suggestion
The maximum value of selected fields for a dataset is of *10 fields per query* in the Real-Time Events GraphQL API. If you surpass the fields' limit, the API will return an error response message informing the issue. If you need to analyze more fields, create different queries.
```

File: src/content/docs/pt-br/pages/api-graphql/limites/index.mdx [hunk 2]
```diff
@@ -18,9 +18,15 @@ Você pode consultar até *10.000 linhas por query*. O campo **limit** na sua qu
...
+## Limite de campos selecionados API GraphQL do Real-Time Metrics
 
-Quando você está consultando campos dos conjuntos de dados, o número máximo de campos selecionados é de *35 campos por query*. Se você ultrapassar o limite de campos, a API exibirá uma mensagem de erro informando o problema.
+O número máximo de campos selecionados para um dataset é de *35 campos por query* na API GQL do Real-Time Metrics. Se você ultrapassar o limite de campos, a API exibirá uma mensagem de erro informando o problema.
+
+---
+
+## Limite de campos selecionados API GraphQL do Real-Time Events
+
+O número máximo de campos selecionados para um dataset é de *10 campos por query* na API GQL do Real-Time Events. Se você ultrapassar o limite de campos, a API exibirá uma mensagem de erro informando o problema. Se você precisar analisar mais campos, crie queries diferentes.
```
Comment (MarianaAguilera) [inline src/content/docs/pt-br/pages/api-graphql/limites/index.mdx:29]:
```suggestion
O número máximo de campos selecionados para um dataset é de *10 campos por query* na API GraphQL do Real-Time Events. Se você ultrapassar o limite de campos, a API exibirá uma mensagem de erro informando o problema. Se você precisar analisar mais campos, crie queries diferentes.
```

**Review by MarianaAguilera — APPROVED** [review]

File: src/content/docs/en/pages/api-graphql/limits/limits.mdx [hunk 3]
```diff
@@ -13,13 +13,19 @@ To keep **GraphQL API** working with enhanced performance, the API has predeterm
 
 ## Query rows limit
 
-You can query up to *10,000 rows per query*. The **limit** field in your query controls this restriction, and you can set it as you desire as long as it doesn’t surpass the **GraphQL API** limit. If you surpass it,  the API will return an Error Response message informing the issue.
+You can query up to *10,000 rows per query*. The **limit** field in your query controls this restriction, and you can set it as you desire as long as it doesn’t surpass the limit. If you surpass it, the API will return an error response message informing the issue.
 
 ---
 
-## Selected fields limit
+## Selected fields limit Real-Time Metrics GraphQL API
```
Comment (hannah-sk) [inline src/content/docs/en/pages/api-graphql/limits/limits.mdx:21]:
```suggestion
```

File: src/content/docs/pt-br/pages/api-graphql/limites/index.mdx [hunk 4]
```diff
@@ -18,9 +18,15 @@ Você pode consultar até *10.000 linhas por query*. O campo **limit** na sua qu
 
 ---
 
-## Limite de campos selecionados
+## Limite de campos selecionados API GraphQL do Real-Time Metrics
```
Comment (hannah-sk) [inline src/content/docs/pt-br/pages/api-graphql/limites/index.mdx:22]:
```suggestion
```

File: src/content/docs/pt-br/pages/api-graphql/limites/index.mdx [hunk 5]
```diff
@@ -18,9 +18,15 @@ Você pode consultar até *10.000 linhas por query*. O campo **limit** na sua qu
...
 
-Quando você está consultando campos dos conjuntos de dados, o número máximo de campos selecionados é de *35 campos por query*. Se você ultrapassar o limite de campos, a API exibirá uma mensagem de erro informando o problema.
+O número máximo de campos selecionados para um dataset é de *35 campos por query* na API GQL do Real-Time Metrics. Se você ultrapassar o limite de campos, a API exibirá uma mensagem de erro informando o problema.
+
+---
+
```
Comment (hannah-sk) [inline src/content/docs/pt-br/pages/api-graphql/limites/index.mdx:26]:
```suggestion
```

File: src/content/docs/pt-br/pages/api-graphql/queries/index.mdx [hunk 6]
```diff
@@ -20,6 +20,10 @@ Você pode usar queries para dois tipos de dados:
 
 Para cada tipo de dados, você consultará com diferentes [conjuntos de dados da GraphQL](/pt-br/documentacao/produtos/graphql-api-recursos/#conjuntos-de-dados).
 
+:::tip[dica]
+Confira os [limites da API GraphqQL](/pt-br/documentacao/produtos/graphql-api-limites/) para garantir que sua query é válida.
```
Comment (hannah-sk) [inline src/content/docs/pt-br/pages/api-graphql/queries/index.mdx:24]:
```suggestion
Confira os [limites da API GraphQL](/pt-br/documentacao/produtos/graphql-api-limites/) para garantir que sua query é válida.
```

**Review by LucasGeneroso — APPROVED** [review]

**Review by peeetry — APPROVED** [review]

File: src/content/docs/en/pages/api-graphql/limits/limits.mdx [hunk 7]
```diff
@@ -13,13 +13,19 @@ To keep **GraphQL API** working with enhanced performance, the API has predeterm
...
 
-When you’re querying for datasets’ fields, the maximum value of selected fields is of *35 fields per query*. If you surpass the fields' limit, the API will return an Error Response message informing the issue.
+The maximum value of selected fields for a dataset is of *35 fields per query* in the Real-Time Metrics GQL. If you surpass the fields' limit, the API will return an error response message informing the issue.
```
Comment (bru-andrade) [inline src/content/docs/en/pages/api-graphql/limits/limits.mdx]:
```suggestion
The maximum value of selected fields for a dataset is of *35 fields per query* in the Real-Time Metrics GraphQL API. If you surpass the limit of fields, the API will return an error response message informing the issue.
```

Mesmo comment da Mari aqui em baixo.

File: src/content/docs/pt-br/pages/api-graphql/limites/index.mdx [hunk 8]
```diff
@@ -18,9 +18,15 @@ Você pode consultar até *10.000 linhas por query*. O campo **limit** na sua qu
 
 ---
 
-## Limite de campos selecionados
+## Limite de campos selecionados API GraphQL do Real-Time Metrics
 
-Quando você está consultando campos dos conjuntos de dados, o número máximo de campos selecionados é de *35 campos por query*. Se você ultrapassar o limite de campos, a API exibirá uma mensagem de erro informando o problema.
+O número máximo de campos selecionados para um dataset é de *35 campos por query* na API GQL do Real-Time Metrics. Se você ultrapassar o limite de campos, a API exibirá uma mensagem de erro informando o problema.
```
Comment (bru-andrade) [inline src/content/docs/pt-br/pages/api-graphql/limites/index.mdx]:
```suggestion
O número máximo de campos selecionados para um dataset é de *35 campos por query* na API GraphQL do Real-Time Metrics. Se você ultrapassar o limite de campos, a API exibirá uma mensagem de erro informando o problema.
```

**Review by bru-andrade — APPROVED** [review]

File: src/content/docs/en/pages/api-graphql/limits/limits.mdx (hunk 7 above)
Comment (hannah-sk) [reply src/content/docs/en/pages/api-graphql/limits/limits.mdx]: é pra ser GQL, mesmo :D

**Pull request merged**